### ISSUE TITLE
Declare variables feeGasPrice and feeGasLimit in Consensus contract

### ISSUE THREAD

**Issue by jayeshbairagi** [OpenST/compute#115]
- Define`uint256 feeGasPrice` and `uint256 feeGasLimit` in `Consensus.sol`
```solidity
uint256 public feeGasPrice = uint256(0);
uint256 public feeGasLimit = uint256(0);
```
- These values will be used while calling `ConsensusGateway::declareOpenKernel`
- These should return constant values
Note: For now the values returned will be `0`. Technical governance will be allowed to update these values later(Not in scope right now).

**Comment by abhayks1** [comment]
@jayeshbairagi Since the values of variables `feeGasPrice` and `feeGasLimit` is constant, I think we should define them as constant. Later when there is setter logic we can make them variables.
```
uint256 public constant FEE_GAS_PRICE = uint256(0);
uint256 public constant FEE_GAS_LIMIT = uint256(0);
``` 
What do you think?

**Comment by jayeshbairagi** [comment]
@abhayks1 Here's what I think regarding this
- These two parameters will be provided and managed by the techgov. The logic regarding updating these values will later be finalized in either setup function or some other way. 
- So, for these parameters which we think will be needed as `storage`, we should keep them as variables rather than constants. Upgradability will be easy this way. If we make these constants right now then we will have to update it again(after implementing logic).

**Comment by abhayks1** [comment]
Thank you for the reply.
Agree!  We can keep these variables. 
As discussed initialization of variables can be done in Consensus.setup function.

**Comment by abhayks1** [comment]
@jayeshbairagi this ticket can be closed. It's implemented in PR #118

**Comment by jayeshbairagi** [comment]
@abhayks1, thank you for informing. Closing this.
